locales/en/plugin__kubevirt-plugin.json

```diff
@@ -876,6 +876,7 @@
   "Terms": "Terms",
   "The changes you have made require this VirtualMachine to be restarted.": "The changes you have made require this VirtualMachine to be restarted.",
   "The channel to track and receive the updates from.": "The channel to track and receive the updates from.",
+  "The DataImportCron polls disk images and imports the images as persistent volume claims. You can configure the image source and other settings on the DataImportCron details page.": "The DataImportCron polls disk images and imports the images as persistent volume claims. You can configure the image source and other settings on the DataImportCron details page.",
```

Comment:
'The DataImportCron...' sounds strange here. I think it should either be 'DataImportCron...' or 'The DataImportCron job...'.

'polls disk images and imports the images as...' sounds redundant. I think 'polls disk images and imports them as...' sounds cleaner.

I believe the convention for CRs is to spell them in camel case, such as `PersistentVolumeClaims`.

Reply:
The message comes from the suggestion of Avital. IMHO he is not wrong, maybe just not 100% correct. And there was another suggestion of Ronen: https://bugzilla.redhat.com/show_bug.cgi?id=2148322#c9
The help text was unfortunately not part of the design and it seems there is no one except Ronen and Avital who could help to formulate the message. It's totally up to us 🤷🏼‍♀️
Maybe we could discuss that also with someone from the doc team? Or just to copy something from the doc? Or you can add also your own suggestion ;) I'd be very happy for that! :)

Reply:
For the first, I would go with 'DataImportCron'. Although "job" makes logical sense, it looks odd after an object.

+1 "imports them"

Yes, CRs are spelled in camel case.